### DISCUSSION TITLE
Observations at time 0 with evid = 4

### DISCUSSION THREAD

**Discussion by BoburaII** [LAPKB/Pmetrics#235] [Q&A]
Hi, 
I’m analyzing a PK dataset with observations recorded at different patient visits (2 and 4 months). Using evid=4 to reset compartments between visits introduces an observation = 0 at time 0 which isn’t in the dataset. Exclusion of the id with this observation results in selection of another id from the second visit with obs = 0 at time 0. Why is this the case?
Thanks.

![popwtresd](https://github.com/user-attachments/assets/8a8e5f22-3a94-4749-90d2-916339cd5526)

**Comment by mnneely** [comment]
It's been a while since I looked at this, but each EVID=4 resets all compartment values and the time to 0, but increments the block value by one. So the first EVID=4 will have observations of zero at [time 0, block 2] for all subjects with such reset events. I suspect what you're seeing with the plot is because the points in your plot at time 0 are exactly superimposed, so by excluding one subject, you reveal the next one in the stack of points at time 0. Plotly only generates a tool tip with information for the topmost layer of superimposed points.

**Reply by BoburaII** [reply]
Hi Michael,
Thanks for your response and explanation.
